addons/sale_mrp/models/sale_mrp.py

```diff
@@ -20,7 +20,7 @@ def _compute_qty_delivered(self):
                 if bom and bom.type == 'phantom':
                     moves = line.move_ids.filtered(lambda m: m.picking_id and m.picking_id.state != 'cancel')
                     bom_delivered = all([move.state == 'done' for move in moves])
```

Comment:
I think it is better to do this :

`bom_delivered = moves and all([move.state == 'done' for move in moves])`

Reply:
Yes, it seems better